gui/src/components/CopyJobSpec.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles';
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+const styles = theme => ({
+  button: {
+    margin: theme.spacing.unit,
+  },
+  inform: {
+	display: "inline-block",
+    opacity: "1",
+  }
+});
+    
+
+class Copy extends React.Component {
```

Comment:
If we use the ripple feedback on the button we can convert this component to a pure function and remove the state management.